### ISSUE TITLE
game crashing, i think when valhallammo trys to drop treasure?

### ISSUE THREAD

**Issue by Neone26** [Athlaeos/ValhallaMMO#32]
The game crashes randomly, tho it only logs me out of the world. The message on the error screen is: 
Internal Exception: java.lang.NullPointerException: Cannot invoke "java.lang.Float.floatValue()" because the return value of "java.util.Map.get(Object)"is null
I've noticed it happens when I'm chopping trees (I think when a golden apple should spawn, because I haven't gotten any, but I've crashed several times), and when I'm digging, tho I have gotten a few things from digging, maybe a specific item is triggering this then?
I am using other plugins alongside ValhallaMMO.

**Comment by Athlaeos** [comment]
Does it still happen after a restart of the server?

**Comment by Neone26** [comment]
Yes, I've restarted several times since the issue first started.

**Comment by Athlaeos** [comment]
Can you try and reproduce this on another server? like a localhost server
